specification/providerhub/resource-manager/Microsoft.ProviderHub/preview/2021-06-01-preview/examples/CheckinManifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "ab7a8701-f7ef-471a-a2f4-d0ebbf494f77",
+    "providerNamespace": "Microsoft.Contoso",
+    "api-version": "2021-05-01-preview",
```

Comment:
@diwudd can you update API version in all the example files?

Reply:
Updated. Can you please take another look?

Reply:
@diwudd - signed off from ARM side.